images/sigstore-scaffolding/tests/rekor-test.tf

```diff
@@ -0,0 +1,62 @@
+resource "random_pet" "test-value" {}
```

Comment:
This is just a copy of what was there before, just trying to segment things somewhat.